src/Psalm/Internal/PropertyMap.php

```diff
@@ -260,7 +260,7 @@
         'field_count' => 'int',
         'host_info' => 'string',
         'info' => 'string',
-        'insert_id' => 'int',
+        'insert_id' => 'int|numeric-string',
```

Comment:
I wonder if this will cause unnecessary noise on common setups. Maximum autoincrement value for MySQL is BIGINT UNSIGNED (2^64-1), while PHP_MAX_INT is signed int64 (2^63-1) (on 64bit platforms).

Reply:
I guess we need the numeric-string case because it depends on a flag wheter the mysql driver passes all data as strings or allows int/float etc

Reply:
See MYSQLI_OPT_INT_AND_FLOAT_NATIVE on https://www.php.net/manual/en/mysqli.options.php

Reply:
Docs say it affects column types. Does it also affect `inser_id`?

Reply:
Didnt tested it but it feels logical to me if it would also affect insert_id

Reply:
I just did, and it doesn't affect the type of `insert_id` property.

Reply:
Thx for testing. I will change to int then